### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Central Certificate Stote_WAT: Tool tip for button  "Go to" is not descriptive.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#140]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #WCAG1.1.1; #C+E-H; #Edge

**Environment Details:**
**URL**: https://manage.iis.net/webserver/central-certificate-store
**Browser Details:** Edge(42.17134.1.0)
Web Accessibility Toolbar

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/.
4.Tab till Web Server and press enter.
5.Tab till Central Certificate Store and press enter.
6.Tab till "Select Folder" button and press enter.
7.Check tool tip for button provided before file path.

**Actual Result:**
Tool tip for button  "Go to" is not descriptive.

**Expected Result:**
Tool tips should be descriptive enough for the user to understand.

**User Impact:** 
If tool tips are not descriptive screen reader will read the same and screen reader users will find difficulty in understanding the purpose of the control.

**Attachment:**
![bug_140_central certificate store_tool tip issue](https://user-images.githubusercontent.com/38523250/44766387-d7370d00-ab76-11e8-82ad-98164a37a741.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
